### PR TITLE
fix: tabs and status bar network context

### PR DESCRIPTION
**Pull request by hanszhoou**
## Description
https://mask.atlassian.net/browse/MF-2889
https://mask.atlassian.net/browse/MF-3032

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/shared/src/UI/components/NetworkTab/index.tsx [hunk 1]
```diff
@@ -7,17 +7,16 @@ import type { NetworkPluginID } from '@masknet/shared-base'
 import { TabContext } from '@mui/lab'
 import { Stack, Tab, Typography } from '@mui/material'
 
-interface NetworkTabProps<T extends NetworkPluginID>
-    extends withClasses<'tab' | 'tabs' | 'tabPanel' | 'indicator' | 'focusTab' | 'tabPaper'> {
-    chains: Array<Web3Helper.Definition[T]['ChainId']>
+interface NetworkTabProps extends withClasses<'tab' | 'tabs' | 'tabPanel' | 'indicator' | 'focusTab' | 'tabPaper'> {
+    chains: Array<Web3Helper.Definition[NetworkPluginID]['ChainId']>
     hideArrowButton?: boolean
+    pluginID?: NetworkPluginID
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/NetworkTab/index.tsx]:
```suggestion
    pluginID: NetworkPluginID
```

File: packages/web3-shared/base/src/specs/index.ts [hunk 2]
```diff
@@ -453,7 +453,7 @@ export interface NonFungibleTokenActivity {
     cover: string
     contract_address: string
     token_id?: string
-    trade_token_logo: string
+    token?: FungibleToken<ChainId, SchemaType>
```
Comment (guanbinrui) [inline packages/web3-shared/base/src/specs/index.ts]: What this token for? A payment token?

File: packages/mask/src/plugins/Trader/apis/trending/index.ts [hunk 3]
```diff
@@ -85,7 +85,10 @@ export async function getNonFungibleTokenActivities(
     chainId: Web3Helper.ChainIdAll,
     contractAddress: string,
     cursor: string,
-): Promise<{ content: NonFungibleTokenActivity[]; cursor: string } | undefined> {
+): Promise<
+    | { content: Array<NonFungibleTokenActivity<Web3Helper.ChainIdAll, Web3Helper.SchemaTypeAll>>; cursor: string }
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/apis/trending/index.ts]: Use Web3Helper.NonFungiblrTokenActivity.

File: packages/web3-shared/base/src/specs/index.ts (hunk 2 above)
Comment (hanszhoou) [reply packages/web3-shared/base/src/specs/index.ts]: Yes, token for payment.

image

File: packages/mask/src/plugins/Trader/apis/trending/index.ts (hunk 3 above)
Comment (hanszhoou) [reply packages/mask/src/plugins/Trader/apis/trending/index.ts]: `NonFungibleTokenActivity` is defined at `packages/web3-shared/base/src/specs/index.ts`.

File: packages/web3-shared/base/src/specs/index.ts (hunk 2 above)
Comment (hanszhoou) [reply packages/web3-shared/base/src/specs/index.ts]: Renamed to `trade_token`

File: packages/mask/src/plugins/Trader/apis/trending/index.ts (hunk 3 above)
Comment (hanszhoou) [reply packages/mask/src/plugins/Trader/apis/trending/index.ts]: Done

**Pull request merged**